airbyte-workers/src/main/java/io/airbyte/workers/temporal/sync/WebhookOperationActivityImpl.java

```diff
@@ -64,14 +72,14 @@ public boolean invokeWebhook(OperatorWebhookInput input) {
     // TODO(mfsiega-airbyte): replace this loop with retries configured on the HttpClient impl.
     for (int i = 0; i < MAX_RETRIES; i++) {
       try {
-        HttpResponse<String> response = this.httpClient.send(requestBuilder.build(), HttpResponse.BodyHandlers.ofString());
+        final HttpResponse<String> response = this.httpClient.send(requestBuilder.build(), HttpResponse.BodyHandlers.ofString());
         LOGGER.debug("Webhook response: {}", response == null ? null : response.body());
         LOGGER.info("Webhook response status: {}", response == null ? "empty response" : response.statusCode());
         // Return true if the request was successful.
-        boolean isSuccessful = response != null && response.statusCode() >= 200 && response.statusCode() <= 300;
+        final boolean isSuccessful = response != null && response.statusCode() >= 200 && response.statusCode() <= 300;
         LOGGER.info("Webhook {} execution status {}", webhookConfig.get().getName(), isSuccessful ? "successful" : "failed");
         return isSuccessful;
-      } catch (Exception e) {
+      } catch (final Exception e) {
         LOGGER.warn(e.getMessage());
```

Comment:
Same comment about potentially missing a call to `ApmTraceUtils.addExceptionToTrace(e);`

Reply:
@benmoriceau I think that we need to do another pass over the activities/workflows.  From what I can see, those appear to be attaching correctly to the traces, but I may be mistaken.  Let's see how the ones added to the workers work out and if we see exceptions that we otherwise are currently not seeing, then I will go ahead and update the temporal workflows/activities as well.